bot/cogs/starboard.py

```diff
@@ -236,14 +258,8 @@ def check(r, u):
         log.debug("Updating existing starboard entry")
 
     async def on_raw_reaction_add(self, payload: RawReactionActionEvent):
-        if payload.guild_id != Guild.id:
-            return log.debug("Reaction was not added in the correct guild.")
-
-        if payload.emoji.name != LVL1_STAR:
-            return log.debug("Invalid emoji was reacted")
-
-        if payload.channel_id == Channels.starboard:
-            return log.debug("Can't star a message on the starboard.")
+        if self.failed_these_checks(payload):
```

Comment:
this function name just doesnt click. what is this actually doing?

Reply:
Checks if the payload contains a star reaction, it's in a guild, the reactor is to be counted and it's not reacted in the starboard channel.

Reply:
What about ```
        if not self.conditions_are_met(payload):
            return``` ?

Reply:
how about `is_relevant_to_starboard`